internal/pkg/cli/pipeline_init.go

```diff
@@ -394,16 +394,16 @@ func (opts *InitPipelineOpts) getGitHubAccessToken() error {
 }
 
 // TODO: nice to have a remote GitHub repo's branch list to offer a select menu.
-func (opts *InitPipelineOpts) getGitHubBranch() error {
-	branch, err := opts.prompt.Get(pipelineEnterGitHubBranchPrompt,
-		"Name of the branch that you wish to use in your GitHub repository. By default the branch name is \"master\".",
+func (opts *InitPipelineOpts) getGitBranch() error {
+	branch, err := opts.prompt.Get(pipelineEnterGitBranchPrompt,
+		"Name of the branch that you wish to use to trigger your pipeline builds. By default the branch name is \"master\".",
```

Comment:
nits: 
1. Can we move this help prompt to the top as a const for consistency?
2. Can we add a new line after the period so that it looks nicer in the terminal?